Jenkinsfile

```diff
@@ -33,6 +33,9 @@ pipeline {
       }
       post {
         always {
+          checkVersions recordIssue: false
+          checkVersions cmd: "-f deploy/application/pom.xml"
+

```

Comment:
I think if someone else reads this Jenkinsfile.
He does not know whats going on here.

I just want to outline the 7 thinkings steps:
"Mhhm. He is checking version two times? One time no issues will be recorded.. hmm.. one time the issues will be recorded but with another maven command?"